### ISSUE TITLE
rror! Failed to install react, react-dom, next, try again.

### ISSUE THREAD

**Issue by foxundermoon** [segmentio/create-next-app#37]
npx create-next-app --example with-material-ui-next   web-app
> Success! Downloaded with-material-ui-next files for web-app

  Installing npm modules:
    react
    react-dom
    next

> Error! Failed to install react, react-dom, next, try again.
undefined

**Comment by haroonabbasi** [comment]
facing the same issue, any solution to it?

**Comment by themeth** [comment]
you should add "sudo"

**Comment by webdif** [comment]
Same problem here.
It does not work, even with `sudo`, so I does not seem to be a permission issue.

@foxundermoon Did you find a workaround?

**Comment by kn3ny** [comment]
First of all, [with-material-ui-next is moved to another repo](https://github.com/zeit/next.js/tree/canary/examples/with-material-ui) now, so the command in the first post will always fail like this:

```
$ npx create-next-app --example with-material-ui-next web-app
> Error! Error downloading with-material-ui-next files for web-app
(node:75227) UnhandledPromiseRejectionWarning: HttpError: {"message":"Not Found","documentation_url":"https://developer.github.com/v3/repos/contents/#get-contents"}
```

---

However, also in another example, this kind of error will reproduce when `yarn install` command failed in the example directory which is downloaded.

```
$ npx create-next-app --example with-firebase-hosting app
...
> Error! Failed to install react, react-dom, next, try again.
undefined
(node:74870) UnhandledPromiseRejectionWarning: Error: yarn installation failed
    at /(snipped)/node_modules/create-next-app/lib/utils/install.js:39:23
```

In my case (`with-firebase-hosting`), I found actual error message by `yarn install` is :
```
error with-firebase-hosting@4.0.1: The engine "node" is incompatible with this module. Expected version "8". Got "10.15.0"
error Found incompatible module
```
In short, the reason is 'engines:{"node": "8"}' is written in the package.json of `with-firebase-hosting` but I run node.js v10.

I think general workaround for this issue is manually download example directory and run npm/yarn install command on your own then fix the problem.

**Comment by rendisimamora** [comment]
I added "sudo npx create-next-app - sample with-material-ui-next web-app" its works

**Comment by luqmanyasin** [comment]
I seem to be having the same issue even when installing without an example script on OSX.

Running the command with `sudo` does not help at all, any suggestions anyone?
